src/main/java/org/openhab/binding/zigbee/converter/color/ColorHelper.java

```diff
@@ -0,0 +1,179 @@
+/**
```

Comment:
I wonder if this shouldn't be merged into the current HSBType class? This already supports some of the conversions you're adding (ie HSB <> RGB) so if we add the XY conversions into this class, to me that makes more sense. If not, we should at least try and reuse the existing conversions that are available.

Reply:
As discussed above, will merge into the HSBType class.